src/tests/ftest/server/metadata.py

```diff
@@ -145,7 +145,7 @@ def test_metadata_fillup(self):
         Use Cases:
             ?
 
-        :avocado: tags=metadata,metadata_fill,nvme,small
+        :avocado: tags=all,metadata,pr,small,metadatafill
```

Comment:
Not related to this line, but with the change in ior_utils.py to use a TestPool object instead of a DaosPool object, the test_metadata_server_restart test will fail.

Updates needed / should be implemented:

L57:
`ior_cmd.daos_cont.update(cont_uuid, "ior.cont_uuid")`
L95-105:
```
self.pool = TestPool(self.context, self.log)
self.pool.get_params(self)
self.pool.create()
```
The last one will also require a tweak to the yaml file: setname -> group.
There are other uses of self.pool in currently skipped tests that could possibly benefit from using TestContainer.